### DISCUSSION TITLE
[Enhancement]:  Auto Sync

### DISCUSSION THREAD

**Discussion by Itsredsect** [ciderapp/Cider#638] [Feature Request]
### The feature you would like

It would be cool if Cider would sync with Apple Music from time to time so that new playlists you create in Apple Music would show up in Cider in a timely manner. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cryptofyre** [comment]
adding to this idea.

cider setting option for automatic playlist refresh interval
